pykoi/retrieval/vectordb/vectordb_factory.py

```diff
@@ -16,14 +16,16 @@ class VectorDbFactory:
 
     @staticmethod
     def create(
-        model_name: Union[str, LlmName], vector_db_name: Union[str, VectorDbName]
+        model_name: Union[str, LlmName], vector_db_name: Union[str, VectorDbName], host: str, port: int
```

Comment:
nit: could you please help update this to use **kwargs for `host` and `port` for epsilla database? This change will require to always pass in host and port for other vector db.